src/Symfony/Bundle/SecurityBundle/Security.php

```diff
@@ -127,7 +128,7 @@ public function login(UserInterface $user, ?string $authenticatorName = null, ?s
 
         $authenticator = $this->getAuthenticator($authenticatorName, $firewallName);
 
-        $this->container->get('security.user_checker')->checkPreAuth($user);
```

Comment:
not sure that this will solve #57706, `security.user_checker` is still bound to `InMemoryUserChecker` as default.

We probably need create service that will be aware about all user_checker's per firewall and based on firewall name, all required service.

Reply:
thanks for commenting
to be franc I've tested several things, see all PR force push
so for now I am quite waiting core devs hints

i do also have created a workaround on my project until core is fixed so for now it really dont bother much in a security POV on my side